Description: Don't link not needed libs.
 Karl want to keep linking these libs.
Author: Norbert Preining <norbert@preining.info>
Last-Update: 2021-06-26
---
 texk/web2c/am/texmf.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texlive-bin-2024.20240313.70630.orig/texk/web2c/am/texmf.am
+++ texlive-bin-2024.20240313.70630/texk/web2c/am/texmf.am
@@ -163,7 +163,7 @@
 # All the x_... and X_... variables will be empty if we aren't supporting X.
 # Follow the library order used in X11R6 itself:
 #   -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 (some may not be present).
-mf_x_libs = $(X_LIBS) $(x_tool_libs) $(X_PRE_LIBS) $(x_ext_lib) $(wlibs) $(X_EXTRA_LIBS)
+mf_x_libs = $(X_LIBS) $(x_tool_libs) $(wlibs) $(X_EXTRA_LIBS)
 endif !WIN32
 
 ## mf and mf-nowin compile mfextra.c with different CPPFLAGS.
